src/Symfony/Bundle/FrameworkBundle/Command/WorkflowDumpCommand.php

```diff
@@ -63,14 +79,11 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $container = $this->getApplication()->getKernel()->getContainer();
         $serviceId = $input->getArgument('name');
 
-        if ($container->has('workflow.'.$serviceId)) {
-            $workflow = $container->get('workflow.'.$serviceId);
+        if (isset($this->workflows['workflow.'.$serviceId])) {
```

Comment:
@StaffNowa as you seem to inject `workflows` like I've show you on https://github.com/94noni/symfony/commit/90752fe9ccc6b6abf2abf9d45f4e4ba566a1ab36

no need here of the prefix `workflow` or `state_machine` this is already the key of this array, aka the `$workflowId` inside the `FrameworkExtension`

Reply:
Hey, Thanks found an issue which not seen yesterday.